root/programs/C#/Samples/WebApp_sample/ProjectX_sample/Framework/Js/common.js

```diff
@@ -143,22 +151,19 @@ function Fx_OnSubmit() {
     // ・ASP.NET Ajax Extension
     // のどちらも、通過する。
 
-    // ・IE6.0のhrefのdoPostBackだと動かない。
-    // ・OperaのhrefのdoPostBack（二重送信時限定）だと動かない。
-
-    // alert(navigator.appName);
-    // alert(navigator.appVersion);
-
     // ----------
 
+    // In Chrome, Safari and Firefox, document.readyState always returns complete
+    // Hence double transmission is prevented using Form_IsSubmitted flag
+
     // ↑Operaの場合、ポスト後直ちにタイマーが無効になるので、実質機能しない。
     //   ※ ただし、Ajaxの場合は、タイマーが無効にならない。
```

Comment:
@daisukenishino 

Does this comment describe that Form_IsSubmitted flag is useless for preventing double transmission?
(You said "Opera の場合、実質機能しない")

But, from line 284 to line 303, you are preventing double transmission using Form_IsSubmitted.

In case of using Opera, is Form_IsSubmitted flag useful? or useless?
Or, is this comment wrong?